cmd/snapshot-controller/main.go

```diff
@@ -59,6 +63,45 @@ var (
 	version = "unknown"
 )
 
+// Checks that the VolumeSnapshot v1 CRDs exist.
+func ensureCustomResourceDefinitionsExist(kubeClient *kubernetes.Clientset, client *clientset.Clientset) error {
+	condition := func() (bool, error) {
+		var err error
+		_, err = kubeClient.CoreV1().Namespaces().Get(context.TODO(), "kube-system", metav1.GetOptions{})
+		if err != nil {
+			// only execute list VolumeSnapshots if the kube-system namespace exists
```

Comment:
If err != nil, that means "kube-system" may not exist.  Why do we go ahead and try to find resources in "kube-system"?

Should line 71 be changed to "err == nil" instead?

Reply:
Good catch! I've opened https://github.com/kubernetes-csi/external-snapshotter/pull/507 to fix this, thanks :)